recurly/__init__.py

```diff
@@ -399,6 +399,8 @@ class BillingInfo(Resource):
         'currency',
         'updated_at',
         'external_hpp_type',
+        'gateway_token',
+        'gateway_code'
```

Comment:
It's useful to add a trailing comma in these kind of lists. python allows it and it prevents issues if you move these around or do a copy paste. So add a comma after `gateway_code`